frappe/commands/site.py

```diff
@@ -388,10 +388,15 @@ def use(site, sites_path='.'):
 
 @click.command('backup')
 @click.option('--with-files', default=False, is_flag=True, help="Take backup with files")
+@click.option('--backup-path-db', default=None, help="Compress private and public files")
```

Comment:
Would like to see `--backup-path` option that would store all backup files here

Reply:
Sounds cool. Added!